x-pack/plugins/security_solution/public/common/components/page/index.tsx

```diff
@@ -34,6 +36,27 @@ export const AppGlobalStyle = createGlobalStyle<{ theme: { eui: { euiColorPrimar
   .euiToolTip {
     z-index: 9950 !important;
   }
+  /*
+      overrides the default styling of euiComboBoxOptionsList because it's implemented
+      as a popover, so it's not selectable as a child of the styled component
+   */
+  .euiComboBoxOptionsList {
+    z-index: 9999;
+  }
+
+  /* ensure elastic charts tooltips appear above open euiPopovers */
+  .echTooltip {
+    z-index: 9950;
```

Comment:
This is on the same level as `.euiToolTip`. Is that intentional?

Reply:
I can't speak to these values. All I did was pull them out from below so that they can be reused. My guess is that it is ok due to how the timeline page overlay was designed (it uses a `z-index` of `4000`)